src/content/guides/compatible-terminals.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Compatible Terminals
+description: Compatible payment terminals.
```

Comment:
@Wulingoh Change: Add more context to this description - it'll be seen in search results and link previews

Something like:
```suggestion
description: A comprehensive list of all of the terminals that Centrapay supports.
```